app/models/user/at_csv.rb

```diff
@@ -0,0 +1,35 @@
+class User
+  class AtCsv
+
+    HEADERS = ['Name', 'E-Mail', 'Created Date', 'Last Activity', 'Plans', 'Current Privileges',  'Active']
+
+    def initialize(users)
+      @users = users
+    end
+    
+    def to_csv
+      CSV.generate(headers: true) do |csv|
```

Comment:
I cannot get this to generate my CSV file. I'm at 20 minutes and counting on my local with 31,500 users when logged in as a super admin.

We should either hide the button for super admins, only allow it to appear and process filtered results, or at least include a message next to the button warning that it may take up to 10+ minutes to generate the CSV.

@stephaniesimms and @sjDCC is downloading the list of all users something either of you would need? The easiest option would be for us to hide this button for super admins (we did something similar on another page if I recall due to the huge size of the download/query).